src/presenters/admin/indexation-permalink-warning-presenter.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Presenter class for the warning that is given when the Category URLs (stripcategorybase) option is touched.
+ *
+ * @package Yoast\YoastSEO\Presenters\Admin
+ */
+
+namespace Yoast\WP\SEO\Presenters\Admin;
+
+use WPSEO_Shortlinker;
+use Yoast\WP\SEO\Helpers\Options_Helper;
+use Yoast\WP\SEO\Presenters\Abstract_Presenter;
+
+/**
+ * Indexation_Permalink_Warning_Presenter class.
+ */
+class Indexation_Permalink_Warning_Presenter extends Indexation_Warning_Presenter {
```

Comment:
I am not sure if extending an existing presenter is good practice. This means that changes made for the other warning are reflected immediately to this presenter. @herregroen Do you have an opinion about this?

Reply:
In this case it seems to be to get the `base_alert` from the parent class which I think is a valid use case.

By extending it we're seeing these two alerts should always look the same, which is completely valid if intentional.

Unit tests should be set up though to ensure there are no unintended changes.